src/us/kbase/workspace/test/database/provenance/ProvenanceTestCommon.java

```diff
@@ -0,0 +1,62 @@
+package us.kbase.workspace.test.database.provenance;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ProvenanceTestCommon {
+	public static final String NS = null;
+	public static final String WHITESPACE = "\n\n    \f     \t\t  \r\n   ";
+
+	public static final List<String> EMPTY_STRINGS = Collections.unmodifiableList(Arrays.asList(
```

Comment:
Source of the use of the word `empty`:
```java
	/** Check if a string is null or whitespace only.
	 * @param s the string to test.
	 * @return true if the string is null or whitespace only, false otherwise.
	 */
	public static boolean isNullOrEmpty(final String s) {
		return s == null || s.trim().isEmpty();
	}
```

I can easily change it to `WHITESPACE`.

Reply:
Changed.

Reply:
yeah, pretty sure that's cruft, I think the `trim()` was added later. Either way it's wrong and at some point I should update it

Reply:
I've renamed the function `isNullOrWhitespace` to be more accurate.

Reply:
Oh great, thanks. I figured there would be  more usages than that.

Reply:
:+1: